docs/source/cli/model-training-inference/configuration-run.rst

```diff
@@ -70,16 +70,34 @@ GraphStorm provides a set of parameters to config the GNN model structure (input
     - Yaml: ``model_encoder_type: rgcn``
     - Argument: ``--model-encoder-type rgcn``
     - Default value: This parameter must be provided by user.
-- **node_feat_name**: User defined feature name. It accepts two formats: a) `fname`, if a node has node features, the corresponding feature name will be fname; b) `ntype0:feat0 ntype1:featA ...`, different node types have different node feature name(s). In the example, "ntype0" has a node feature named "feat0" and "ntype1" has a node feature named "featA".
+- **node_feat_name**: User defined feature name. It accepts two formats like: a) `fname`, if all node types have the same node feature name, the corresponding feature name will be `fname`; b) `ntype0:feat0 ntype1:featA ...`, different node types have different node feature name(s). In the below example, `ntype0` has a node feature named `feat0` and `ntype1` has two node features named `featA` and `featB`.
 
     - Yaml: ``node_feat_name:``
-                | ``- "ntype1:featA,featB"``
                 | ``- "ntype0:feat0"``
+                | ``- "ntype1:featA,featB"``
     - Argument: ``--node-feat-name "ntype0:feat0 ntype1:featA,featB"``
     - Default value: If not provided, there will be no node features used by GraphStorm even graphs have node features attached.
 
     .. Note:: Characters ``:`` and white space are not allowed to be used in node feature names.  And in Yaml format, need to put each node's feature in a separated line that starts with a hyphon.
 
+- **edge_feat_name**: User defined edge feature name. It accepts two formats like: a) `fname`, if all edge types have the same feature name, the corresponding feature name will be `fname`; b) `src_ntype1,etype1,dst_ntype1:feat0,... src_ntype2,etype2,dst_ntype2:feata ...`, if different edge types have different feature name(s). In the below example, `src_ntype1,etype1,dst_ntype1` edge type has two features named `feat0`, and `feat1`, and `src_ntype2,etype2,dst_ntype2` edge type has one feature named `feata`.
+
+    - Yaml: ``edge_feat_name:``
+                | ``- "src_ntype1,etype1,dst_ntype1:feat0,feat1"``
+                | ``- "src_ntype2,etype2,dst_ntype2:feata"``
+    - Argument: ``—-edge-feat-name src_ntype1,etype1,dst_ntype1:feat0,feat1 src_ntype2,etype2dst_ntype2:feata``
+    - Default value: If not provided, there will be no edge features used by GraphStorm even graphs have edge features attached.
+
+    .. Note:: In the latest version, the RGCN encoder has been modified to support using edge features during message passing computation. If users would like to use edge features, please set the ``model_encoder_type`` to be ``rgcn``.
```

Comment:
Shall we also explain what the outcomes will be if ``model_encoder_type`` is not set to ``rgcn`` when using edge features?

Reply:
Updated with additional sentence, telling users there will be an assertion error.